Remotion/Core/Collections.Caching.UnitTests/NullCacheTest.cs

```diff
@@ -36,8 +36,7 @@ public void SetUp ()
     [Test]
     public void TryGetValue ()
     {
-      object actual;
-      Assert.That (_cache.TryGetValue ("anyKey", out actual), Is.False);
+      Assert.That (_cache.TryGetValue ("anyKey", out var actual), Is.False);

```

Comment:
Same question as above - `out var` vs declaration. Multiple occurrences in this file.